src/System.Windows.Forms/src/System/Windows/Forms/DataGridViewCheckBoxCell.cs

```diff
@@ -398,6 +398,23 @@ internal bool ThreeStateInternal
         }
     }
 
+    internal CheckState CheckState
```

Comment:
```suggestion
    private CheckState CheckState
```

Private properties are accessible from the nested classes.